test/util/aggregate-error.test.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from 'chai'
+import { describe, it } from 'mocha'
+
+import {
+  AggregateError,
```

Comment:
This `AggregateError` util would be useful in the GUI and plugins. Do we have a `edge-utils` repo?